src/api/Annotations.js

```diff
@@ -309,7 +309,7 @@ export default class Annotations extends MarkerBasedApi {
             data: { data: { message } },
             errorCallback,
             successCallback,
-            url: this.getUrlWithRepliesForId(annotationId),
+            url: `${this.getUrlWithRepliesForId(annotationId)}?file_id=${fileId}`,
```

Comment:
Nah, same error:
```
"Query parameter 'file_id' is required"
```
Tried it with both
```
data: { data: { message }, file_id: fileId }
```
and
```
 data: { data: { message, file_id: fileId } }
```
just in case. We could look up where that endpoint is defined, but of course lowest LOE here is just to give it the query param that it's asking for.

Reply:
since you received the error `"Query parameter 'file_id' is required"`, does that means this POST previously wasn't working?